components/tag/selectableList/src/index.js

```diff
@@ -0,0 +1,109 @@
+import React, {Component, PropTypes} from 'react'
+import TagSelectable from '@schibstedspain/sui-tag-selectable'
+
+const removeFromArray = (arr, value) => {
+  return arr.filter(item => value !== item)
+}
+
+export default class TagSelectableList extends Component {
+  state = {
+    selectedValues: []
+  }
+
+  constructor (...props) {
+    super(...props)
+    this.tagHandlers = this.props.tagsList.map((tag) => this.toggleValue.bind(this, tag.value))
```

Comment:
If you want to avoid this bind (slow)

you can define toggleValue with fat arrow and it will be bind to this by defaullt.

```
const toggleValue = (value) => { ... }
```

And if you need to return a function instead of the computation result, use currying:

```
const toggleValue = (value) => () => { ... }
```